pkg/component/shoot/system/system.go

```diff
@@ -544,7 +544,8 @@ func viewerClusterRoleBindings() []client.Object {
 
 // ClusterRoleBindings is function generating the shoot ClusterRoleBindings related
 // to the AdminKubeconfig and ViewerKubeconfig deployed in the shoot cluster.
-// Deprecated: Do not use, this function is deprecated and will be removed after v1.127.0 has been released
+// Deprecated: Do not use, this function is deprecated and will be removed after v1.128.0 has been released.
```

Comment:
I've noticed that an empty separation line is usually being added between _docstring_(s) and _deprecation note_(s), do you think it's applicable to include one here as well ?

```suggestion
//
// Deprecated: Do not use, this function is deprecated and will be removed after v1.128.0 has been released.
```